### ISSUE TITLE
Server restart every day

### ISSUE THREAD

**Issue by L3o-pold** [ONLYOFFICE/DocumentServer#761]
Every day in the `/var/log/supervisor/supervisord.log` I see that the server restart.
Using `onlyoffice/documentserver-de:5.4.1.39`

```
2020-01-27 06:35:54,187 WARN received SIGTERM indicating exit request
2020-01-27 06:35:54,188 INFO waiting for metrics, spellchecker, gc, docservice, converter to die
2020-01-27 06:35:54,190 INFO stopped: spellchecker (terminated by SIGTERM)
2020-01-27 06:35:54,190 INFO stopped: metrics (exit status 0)
2020-01-27 06:35:54,191 INFO stopped: converter (terminated by SIGTERM)
2020-01-27 06:35:54,195 INFO stopped: docservice (terminated by SIGTERM)
2020-01-27 06:35:54,234 INFO stopped: gc (terminated by SIGTERM)
2020-01-27 06:35:59,271 CRIT Supervisor running as root (no user in config file)
```

```
2020-01-26 06:32:48,147 WARN received SIGTERM indicating exit request
```

```
2020-01-25 06:40:51,297 WARN received SIGTERM indicating exit request
```

`cat /etc/apt/apt.conf.d/20auto-upgrades`

```
APT::Periodic::Update-Package-Lists "0";
APT::Periodic::Unattended-Upgrade "0";
```

Any idea?

**Comment by hahaxd35** [comment]
hope i can help:

> @ShockwaveNN Yeah I'm not sure reloading the server, is a good idea, cause it'll break connections...

Run service nginx reload or /etc/init.d/nginx reload

If you have pending requests, then there will be lingering nginx processes that will handle those connections before it dies, so it's an extremely graceful way to reload configs.

> but currently do not how to elegant solve this problem
mayby you have a look @ rsylog:
http://jensd.be/372/linux/write-to-and-manage-syslog-messages-with-logger-and-rsyslog

after logrotate you only need to reload rsylog (by default after logrote)

**Comment by L3o-pold** [comment]
I don’t think that an ideal solution because even using graceful stop on Nginx, document server are using websocket and it’ll disconnect the user.

**Comment by Githopp192** [comment]
i think .. i'm running into the same issue -->

2020-02-18 06:28:16,759 WARN received SIGTERM indicating exit request
2020-02-18 06:28:16,939 INFO waiting for metrics, spellchecker, gc, docservice, converter to die
2020-02-18 06:28:18,823 INFO stopped: gc (terminated by SIGTERM)
2020-02-18 06:28:21,345 INFO waiting for metrics, spellchecker, docservice, converter to die
2020-02-18 06:28:21,414 INFO stopped: spellchecker (terminated by SIGTERM)
2020-02-18 06:28:21,414 INFO stopped: docservice (terminated by SIGTERM)
2020-02-18 06:28:21,415 INFO stopped: converter (terminated by SIGTERM)
2020-02-18 06:28:22,730 INFO stopped: metrics (exit status 0)

/etc/crontab -->

# m h dom mon dow user  command
25 6    * * *   root    test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.daily )

Three minutes later OnlyOffice will stop working .. every day .. same time
